commit d24458e3055a2066830f7dd274e0966178e98b60 (refs/changes/99/2399/2)
Author: Jan Wyszynski <jan.wyszynski@joyent.com>
Date:   2017-08-15T21:16:31+00:00 (2 years, 2 months ago)
    
    joyent/node-manta#318: node-manta nodejs version support
    Reviewed by: Angela Fong <angela.fong@joyent.com>
    Reviewed by: Dave Pacheco <dap@joyent.com>

diff --git a/README.md b/README.md
index e581700..824635e 100644
--- a/README.md
+++ b/README.md
@@ -168,3 +168,15 @@ Here is how to cut a release:
 
    This will run a couple checks (clean working copy, versions in package.json
    and CHANGES.md match), then will git tag and npm publish.
+
+# Supported Node.js Versions
+
+Currently, node-manta officially supports functionality covered by all but the MPU tests
+that live in the test directory at the project root when used with the following versions
+of node.js:
+
+* v0.10.48
+* v0.12.18
+* v4.8.4
+* v6.11.2
+* v8.3.0
